Jenkinsfile

```diff
@@ -53,6 +53,23 @@ pipeline {
             }
             post { always { runShell("rm -rf ./*") }}
         }
+        stage("Code formatting") {
+            agent {
+                dockerfile {
+                    filename 'docker/debian/Dockerfile'
+                    //Forces image to ignore entrypoint
+                    args "-u root --entrypoint=\'\'"
+                }
+            }
+            steps {
+                sh 'printenv'
+                runShell("""
+                    eval \$(opam env)
+                    make format
```

Comment:
Consider printing a message about:
1. That this is a formatting error that they can resolve with "make format"
2. They should install the git hook

Reply:
I did that initially, but make format stops this runShell. I can work around that though.

Reply:
Can you use `make format || echo "<message>"`?

Reply:
Took me way too many tries but this message is now printed.

The source code was not formatted. Run 'make format | dune promote' and push the changes.
Please consider installing a pre-commit git hook for formatting with the above command.

https://jenkins.mc-stan.org/blue/organizations/jenkins/stanc3/detail/PR-588/14/pipeline

Copy-edit on the message welcome as always.